### DISCUSSION TITLE
أقتراحات | ما يطلبه المتبلوثون هع هع

### DISCUSSION THREAD

**Discussion by MutlaqAldhbuiub** [sudogroup/SudoBot#53] [1- العامة]
![](https://p4.storage.canalblog.com/46/81/943421/71584103_p.gif)
💕🍃آسعــد اللـــه صباحكم///////مسائكم بكل خير وسعادة🍃💕.
اليوم نبي استشارتكم يا منتدى سودوميز ![](https://vb.tgareed.org/attachments/22790/)

بحكم ان الصلاحيات في #6 معقدة نوعا ما ![](https://vb.tgareed.org/attachments/22799/)
فا فكرنا اننا نخليها زي نظام GNU/LINUX ![](https://vb.tgareed.org/attachments/22800/)
المشكلة اني حسيت إنها معقدة أكثر ! ![](http://www2.0zz0.com/2010/07/25/08/496700298.png)
لان نظام التصاريح في جنو/لينكس مبني على ارقام تقريبا
مثل 
777 = كل الصلاحيات 
فااااا جاي هنا في منتداكم الموقر اطلب مشورتكم ![](http://www2.0zz0.com/2010/07/25/08/500996098.png)
الاااان الي صايرررر ~~~~
777 = مووودييير كبييير
755 = نص مودير كبييير 
744 = مشرف قسم
500 = عضو طبيعي

نجيي الان لطريقة الاستخدام ![](http://www.rooh5.com/vb/images/smilies/005.gif)
```
!sudo newrole [role_name] [permissions_number]
```
مثال حقيقي نبي نحط رول جديد اسمه a7laMontada 
ونبي نخليهم مدرراءءء كبباررر ![](http://i892.photobucket.com/albums/ac129/LoveIsTrick/Smiles/GreenMan/uto_2061.gif)

```
!sudo newrole a7laMontada 777
```

عطووووونااا رايكم نبي نتجالد هنا على شان نطلع بإفضل طرريييقه ![](http://i892.photobucket.com/albums/ac129/LoveIsTrick/Smiles/happy/915.gif)

قبل ما انهي الموضووووع 
اهدييييكم اغنية السندباد راشد الماجد
[الاااا ياوقت](https://www.youtube.com/watch?v=PiyNIpCjjy8)

وهنااا ناصل لنهاية الموضووووع يعطييييكم العافيه ![](http://i892.photobucket.com/albums/ac129/LoveIsTrick/Smiles/happy/76.gif)

![](http://jnoon-q.com/vb/image.php?u=1&dateline=1603959149)

**Comment by ma7dev** [comment]
استرجل ياورع وحل المشاكل مزبوط لا اجي اتوطاك

**Reply by ma7dev** [reply]
btw, @MutlaqAldhbuiub wrote this and I copied-and-pasted here. :(

**Comment by mo9a7i** [comment]
ماتوقع برمشنز الغرف مفروض بكون مرتبط بالرولات

تضيف رول جديد
بعدين تخصص برمشنز لكل غرفة او قروب غرف

مثل القروب اي دي والقايل برمشن
هذا شي وهذا شي

**Reply by ma7dev** [reply]
I think these are just general roles for the entire server. Maybe, we will need to add a flag to assign permissions for certain channels/categories, or we need to add a command to edit channels/categories to add additional roles with their permissions.
